gsy_framework/data_classes.py

```diff
@@ -388,6 +392,11 @@ def is_offer_trade(self) -> bool:
         """Check if the instance is an offer trade."""
         return isinstance(self.offer_bid, Offer)
 
+    @property
+    def trade_rate(self):
+        """Return the energy rate of the trade."""
+        return round(self.trade_price / self.traded_energy, 8)
```

Comment:
Please reuse the `DEFAULT_PRECISION` constant. Thanks!
```suggestion
        return round(self.trade_price / self.traded_energy, DEFAULT_PRECISION)
```